Terraform/stag/terragrunt.hcl

```diff
@@ -2,14 +2,19 @@ include {
   path = find_in_parent_folders()
 }
 
-generate "sandbox_tfvars" {
+generate "stag_tfvars" {
   path              = "dev.auto.tfvars"
   if_exists         = "overwrite"
   disable_signature = true
   contents          = <<-EOF
-  environment = "dev"
   fargate_cpu = 512
   fargate_memory = 1024
   app_port = 21000
+  fam_console_idp_name = "PROD-IDIR"
+  application = "phlat-stage"
```

Comment:
this is not affected right? 
application = "phlat-stage"

Reply:
This one may need to change back as well. It's being use quite a bit. It's possible it's able to rename though as this doesn't impact important things like the state. For simplicity you could change it back just to get this running and then change it to "stage" again in the future (when you can do more stable terraform plans)